### PR TITLE
Merge hybrid branch into master but flag with --remote

### PR DESCRIPTION
**Pull request by tobespc**
This is all the latest hybrid changes to be merged into master.   By default, the hybrid changes are off and need to be enabled by running either 

sh run.sh --remote   or
sh start.sh --remote

### CODE REVIEW

File: src/pfe/portal/package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "./npm-start.sh",
-    "prod": "node server.js",
-    "dev": "nodemon server.js",
+    "prod": "node --inspect=0.0.0.0:9777 server.js",
+    "dev": "node --inspect=0.0.0.0:9777 server.js",
```
Comment (hhellyer) [inline src/pfe/portal/package.json]: I'm assuming this is enabling the debugger. Do we want to leave debug on and available on a port in production? (Not sure if running node with `--inspect` has a performance impact.)

File: src/pfe/portal/routes/projects/remoteBind.route.js [hunk 2]
```diff
@@ -175,11 +197,43 @@ router.post('/api/v1/projects/:id/remote-bind/clear', async (req, res) => {
 
       log.info(`Removing locally deleted files from project: ${project.name}, ID: ${project.projectID} - ` +
         `${filesToDelete.join(', ')}`);
-
+      // remove the file from pfe container
       await Promise.all(
-        filesToDelete.map(oldFile => exec(`rm -rf ${path.join(pathToClear, oldFile)}`))
+        filesToDelete.map(oldFile => exec(`rm -f ${path.join(pathToClear, oldFile)}`))
```
Comment (hhellyer) [inline src/pfe/portal/routes/projects/remoteBind.route.js]: Without `-r` this won't delete directories.

**Comment by hhellyer** [comment]
@tobespc - Please squash before merging!

File: src/pfe/portal/package.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "./npm-start.sh",
-    "prod": "node server.js",
-    "dev": "nodemon server.js",
```
Comment (ssh24) [inline src/pfe/portal/package.json]: Removing this also removed the dev environment on whole PFE. The dev environment was useful to easily push changes without rebuilding PFE. https://github.com/eclipse/codewind/issues/695

**Pull request merged**